main.go

```diff
@@ -112,6 +112,11 @@ func (t *torcher) createAndRunApp() {
 			Usage: "file path of raw binary profile; alternative to having go-torch query pprof endpoint " +
 				"(binary profile is anything accepted by https://golang.org/cmd/pprof)",
 		},
+		cli.StringFlag{
```

Comment:
I would make this optional. When we give raw pb.gz files, the binary name is not needed.